fakestorage/response.go

```diff
@@ -119,6 +120,15 @@ type objectResponse struct {
 	Metadata        map[string]string      `json:"metadata,omitempty"`
 }
 
+func newProjectedObjectResponse(obj ObjectAttrs, projection storage.Projection) objectResponse {
+	objResponse := newObjectResponse(obj)
+	if projection == storage.ProjectionNoACL {
+		objResponse.ACL = nil
+		// TODO: `objResponse.Owner = nil` once we have the `Owner` field in `objectResponse`
```

Comment:
when resolving the merge conflict, can you also address this TODO?

Reply:
I have just removed the TODO instead.

It was probably too speculative to include it to begin with, as it's not really actionable at the moment. Only once (if ever) we implement support for the `owner` attribute in the object response.